src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin-left: 50px;
+  }
+}
+
+.nav {
+  display: flex;
+  height: inherit;
+  margin: 50px;
```

Comment:
Put each BEM-block in a new file. 
Don't give margins to BEM-block

Reply:
fix it everywhere

Reply:
Fixed eternal positioning and divided blocks on different files.